pandas_gbq/gbq.py

```diff
@@ -617,23 +619,18 @@ def _bqschema_to_nullsafe_dtypes(schema_fields):
     See: http://pandas.pydata.org/pandas-docs/dev/missing_data.html
     #missing-data-casting-rules-and-indexing
     """
+    import db_dtypes
+
     # If you update this mapping, also update the table at
     # `docs/reading.rst`.
     dtype_map = {
-        "DATE": "datetime64[ns]",
-        "DATETIME": "datetime64[ns]",
         "FLOAT": np.dtype(float),
-        "GEOMETRY": "object",
```

Comment:
I'm not sure how the changes here to non-datetime-related mappings relates to this PR.

If these changes are intentional, then the comment above seems to require a corresponding update to `docs/reading.rst`.

Reply:
`object` types were removed because it's the default anyway.